pkg/internal/dkgtest/dkgtest.go

```diff
@@ -116,7 +116,7 @@ func executeDKG(
 	}
 
 	resultSubmissionChan := make(chan *event.DKGResultSubmission)
-	chain.ThresholdRelay().OnDKGResultSubmitted(
+	_, _ = chain.ThresholdRelay().OnDKGResultSubmitted(
```

Comment:
Let's drop `error` from the returned parameters of `OnDKGResultSubmitter`, just like we did for all functions of `RelayEntryInterface`.